releases.md

```diff
@@ -25,5 +25,34 @@ An Ignition release follows the form "Ignition Codename", for example Ignition A
 
 ## Library Versions
 
+Ignition adheres to [semantic versioning](https://semver.org), with the
+addition that ABI is considered part of the public interface. In summary:
+
+* **Major** version increased when incompatible ABI/API changes are made.
+* **Minor** version increased when functionality has been added in a
+  backwards-compatible manner, but it may not be forward-compatible. That is,
+  code compiled against minor X will work with minor Y, but code compiled
+  against minor Y may not work with minor X.
```

Comment:
it would be clearer to me to use `N` and `N+1` instead of `X` and `Y`

Reply:
Done in  f3665ff